server/routes/visitors.test.ts

```diff
@@ -477,56 +467,7 @@ describe('GET /visit/additional-support/:offenderNo', () => {
 })
 
 describe('POST /visit/additional-support/:offenderNo', () => {
-  beforeAll(() => {
-    const visitorList: VisitorListItem[] = [
-      {
-        personId: 4321,
-        name: 'Jeanette Smith',
-        dateOfBirth: '1986-07-28',
-        adult: true,
-        relationshipDescription: 'Sister',
-        address:
-          'Premises,<br>Flat 23B,<br>123 The Street,<br>Springfield,<br>Coventry,<br>West Midlands,<br>C1 2AB,<br>England',
-        restrictions: [],
-      },
-      {
-        personId: 4322,
-        name: 'Bob Smith',
-        dateOfBirth: '1986-07-28',
-        adult: true,
-        relationshipDescription: 'Brother',
-        address: '1st listed address',
-        restrictions: [],
-      },
-      {
-        personId: 4323,
-        name: 'Ted Smith',
-        dateOfBirth: '1968-07-28',
-        adult: true,
-        relationshipDescription: 'Father',
-        address: '1st listed address',
-        restrictions: [],
-      },
-      {
-        personId: 4324,
-        name: 'Anne Smith',
-        dateOfBirth: '2018-03-02',
-        adult: false,
-        relationshipDescription: 'Niece',
-        address: 'Not entered',
-        restrictions: [],
-      },
-      {
-        personId: 4325,
-        name: 'Bill Smith',
-        dateOfBirth: '2018-03-02',
-        adult: false,
-        relationshipDescription: 'Nephew',
-        address: 'Not entered',
-        restrictions: [],
-      },
-    ]
-    prisonerVisitorsService = new MockPrisonerVisitorsService()
+  beforeEach(() => {
```

Comment:
Had to switch this back to a `beforeEach()` as otherwise `visitSessionData` wasn't 'clean' before each test. Cut out the `VisitorList` and associated service as not needed for this route.